modules/tensor_mechanics/src/actions/DomainIntegralAction.C

```diff
@@ -118,7 +129,7 @@ DomainIntegralAction::DomainIntegralAction(const InputParameters & params)
                                       : 0.0),
     _treat_as_2d(getParam<bool>("2d")),
     _axis_2d(getParam<unsigned int>("axis_2d")),
-    _convert_J_to_K(false),
+    _output_type(getParam<MooseEnum>("output_type")),
```

Comment:
I just realized something. I guess this is only applicable if you're computing J. It's also kind of weird that if you want C, you have to ask for the J integral and then set output_type if you want C.  What if we got rid of this option, and instead added 'C_integral' and 'K_from_J_integral' as options for 'integrals'?

Reply:
Should be done now. `CIntegral`, `JIntegral`, and `KFromJIntegral` are explicitly exclusive in the action with the latest changes.